### ISSUE TITLE
Updating Files

### ISSUE THREAD

**Issue by tkhyn** [tkhyn/dirsync#4]
**[Original report](https://bitbucket.org/tkhyn/dirsync/issue/4) by Anonymous.**

----------------------------------------

Is it possible to disable the update function if the file already exists ?

I have the issue that the timestamps are not synced and then it updates every single file even if I only want sync the new files.

--diff shows me that it only wants to update the new ones but sync actually does sync every file again.

I think the line of importance for me would be: syncer.py L.56 >> self._updatefiles = True

But I haven't tested it yet.

In short:

I got 5 files (dir1) for example and 3 files (dir2) it will not update only the 2 new ones because of the different timestamps. (maybe an option to ignore the timestamps is possible) 

Would be great to see an update here because I want to use this script to sync my hard drives.
